### PR TITLE
Change version numbers to 5.2.4

### PR DESCRIPTION
**Pull request by tst-rwildes**
Fixes issue #889

### CODE REVIEW

File: docs/geedocs/5.2.4/answer/7160003.html [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+  <p class="sidebar-toc"><a href="../answer/4556704.html">GEE 5.0.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/3424202.html">GEE 5.0</a></p>
+</div>
+<div class="content">
+<a name="top_of_file"></a>
+<p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
+<h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
+<h2>Release notes: Open GEE 5.2.3</h2>
+
+<p>Open GEE 5.2.3 is currently in development</p>
```
Comment (tst-blaird) [inline docs/geedocs/5.2.4/answer/7160003.html]: I know this is a new file in the 5.2.4 directory but this line in the 5.2.3 directory says "currently in beta testing".  Was it intended to change it to "currently in development" in the 5.2.4 directory?

File: docs/geedocs/5.2.4/answer/7160003.html [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+    <p><strong>Remove TCP/IP connections to PostgreSQL</strong>. With release 5.2.3 Open GEE code will now connect to PostgreSQL exclusively using Unix domain sockets. This will make it easier to secure PostgreSQL for security sensitive installations </p>
+    <p><strong>Volume deletion</strong>. <code>geconfigureassetroot</code> now supports the option <code>--removevolume</code> for volume deletion.</p>
+    <p><strong>Performance tuning for gecombineterrain</strong>. <code>gecombineterrain</code> now supports the option <code>--numCompressThreads</code> which allows configuration of the threads used for operation.  If not specified <code>gecombineterrain</code> defaults this option to match the number of available CPUs.</p>
+    <p><strong>Package name customization at build time</strong>. Scons build now accepts a custom label argument which is appended to the version in RPM file names.</p>
+    <p><strong>Rewrite KML URLs</strong>. Users can rewrite the URLs for KML resorces included in databases at publish time.</p>
```
Comment (tst-blaird) [inline docs/geedocs/5.2.4/answer/7160003.html]: I think a line was removed here that was in the 5.2.3 directory version of this file.  The line in my diff shows as:
<     <p><strong>Maps API Javascript Files</strong>. Maps API Javascript files are now available and installed at
 <code>/opt/google/gehttpd/htdocs/maps/mapfiles</code>.</p>

**Comment by tst-rwildes** [comment]
Yes I think it was intended to be a dummy commit.  Another way to do a
dummy commit is just to use the git commit with --allow-empty and just add
a comment with the -m parameter.

`git commit --allow-empty -m "creating initial branch for xxx"`

**Comment by tst-rwildes** [comment]
Never mind wrong thread...

File: docs/geedocs/5.2.4/answer/7160003.html (hunk 2 above)
Comment (tst-rwildes) [reply docs/geedocs/5.2.4/answer/7160003.html]: Good catches.  They should be fixed now.

File: docs/geedocs/5.2.4/answer/7160003.html [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+            <td>Update geconfigurepublishroot to fully correct file permissions</td>
+            <td>geconfigurepublishroot recursively sets the owner and permissions for publish root</td>
+          </tr>
+          <tr>
+            <td>731</td>
+            <td>Error in publish of SSL-enabled database</td>
+            <td>Temporary fix from 5.2.2 was found to be the best solution</td>
+          </tr>
+          <tr>
+            <td>762</td>
```
Comment (tst-blaird) [inline docs/geedocs/5.2.4/answer/7160003.html]: Sorry, one more question.  It looks like issue 762 is in this file but not in the same file in the 5.2.3 directory.  I think from looking at issue #762 that it is resolved in 5.2.3.  Should this change also be made to the same file in the 5.2.3 directory or is that being done under a different issue?

Comment (tst-rwildes) [reply docs/geedocs/5.2.4/answer/7160003.html]: I think I got this one fixed up too now

**Comment by tst-rwildes** [comment]
@tst-nfarah I made the same change to the 5.2.3 release notes file in the new 5.2.4 folder as what is in #886.  The 5.2.3 release notes file in the 5.2.3 folder will get the needed change when that PR is merged and this PR will take care of the 5.2.4 folder change.  So there should be no conflict and we could merge this now.

Any future changes to 5.2.3 release notes will then have to be changed in both places because we do this copy with each release.  Maybe in the future we need to figure out why we need to do this copying of release folders...

**Pull request merged**